rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -0,0 +1,76 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_ansible
+
+%global foreman_dir /usr/share/foreman
+%global foreman_bundlerd_dir %{foreman_dir}/bundler.d
```

Comment:
These two foreman* lines should be removed as you're using foreman-plugin which provides them.